.github/workflows/main.yml

```diff
@@ -112,10 +112,17 @@ jobs:
         run: |
           export $(grep -v '^#' ./deployment/ci.env | xargs)
           echo "FULLNODE_VERSION=$FULLNODE_VERSION" >> $GITHUB_ENV
+      - name: Check if ci.env changed
+        id: changed-files
+        uses: tj-actions/changed-files@db5dd7c176cf59a19ef6561bf1936f059dee4b74
+        with:
+          files: |
+            deployment/ci.env
       - name: Trigger deployment event ${{ github.ref }}
         env:
           NAMESPACE: "ng-mardunet"
           EVENT_TYPE: "ng_babylon_mardunet"
+        if: steps.changed-files.outputs.any_changed == 'true'
```

Comment:
You need to still deploy , but only gateway. The value on the respository dispatch should change and so then the workflows on radixdlt-iac needs to be udpated